### ISSUE TITLE
sbt warnings for scalaMajorVersion

### ISSUE THREAD

**Issue by ashawley** [typelevel/scalacheck#701]
```
[warn] there are 2 keys that are not used by any other settings/tasks:
[warn]  
[warn] * bench / scalaMajorVersion
[warn]   +- scalacheck/build.sbt:19
[warn] * scalacheck / scalaMajorVersion
[warn]   +- scalacheck/build.sbt:5
```

**Comment by ashawley** [comment]
There's also the following after, but I have no idea if it's related or not:
```
[warn] note: a setting might still be used by a command; to exclude a key from this `lintUnused` check
[warn] either append it to `Global / excludeLintKeys` or call .withRank(KeyRanks.Invisible) on the key
```

**Comment by SethTisue** [comment]
(yes, that's advice about how perhaps to deal with the first message. there is also a possibility of false positives, since these warnings are new)

**Comment by larsrh** [comment]
Fixed by #786
